<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSLSetupWidget</class>
 <widget class="QWidget" name="LSLSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TMSi EEG Connector Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="m_qLabel_Headline">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Available LSL Streams</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listLSLStreams"/>
   </item>
   <item>
    <widget class="QPushButton" name="refreshAvailableStreams">
     <property name="text">
      <string>Refresh available streams</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Stream to connect to:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentStreamDescription">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;None&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output block size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="blockSizeEdit">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
